frontend/packages/multicluster-sdk/src/api/useIsFleetPrometheusAvailable.ts

```diff
@@ -0,0 +1,46 @@
+/* Copyright Contributors to the Open Cluster Management project */
+import { UseIsFleetPrometheusAvailable } from '../types'
+import { useMemo } from 'react'
+import { MultiClusterObservabilityKind } from '../internal/models'
+import { useK8sWatchResource, K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk'
+
+// Define the condition type for proper TypeScript typing
+type K8sCondition = {
+  type: string
+  status: string
+  lastTransitionTime?: string
+  reason?: string
+  message?: string
+}
+
+// Define the MultiClusterObservability resource type
+type MultiClusterObservabilityResource = K8sResourceCommon & {
+  status?: {
+    conditions?: K8sCondition[]
+  }
+}
+
+/**
+ * Hook that determines if the fleet Prometheus is available.
+ *
+ * Checks if a MultiClusterObservability resource has been installed which supports using Thanos to collect
+ * Prometheus metrics data from fleet clusters usch as CPU and Memory.
+ *
+ * @returns `true` if a MultiClusterObservability resource has been installed and is available; `false` otherwise
+ */
+export const useIsFleetPrometheusAvailable: UseIsFleetPrometheusAvailable = () => {
+  const [mcos, loaded, error] = useK8sWatchResource<MultiClusterObservabilityResource[]>({
+    groupVersionKind: MultiClusterObservabilityKind,
+    isList: true,
+  })
```

Comment:
@jeswanke Is there a reason you chose to check this resource? The existing check for this looks for a specific ClusterManagementAddon instead:
https://github.com/stolostron/console/blob/main/frontend/src/atoms.ts#L198-L203

@zlayne Any thoughts on whether these are equivalent? I suppose neither guarantees that Thanos multicluster observability is working correctly. 

Secondly, this is not an RBAC-friendly approach, meaning if a user does not have access to view the `MultiClusterObservability` resource, they will not be able to use fleet prometheus queries. I would recommend adding something to the `/multicloud/hub` API to indicate if fleet observability is enabled. Our ServiceAccount has permission to see the ClusterManagementAddOns already, so that might be the easier path.